spec/services/hyrax/thumbnail_path_service_spec.rb

```diff
@@ -3,6 +3,35 @@
 
   subject { described_class.call(object) }
 
+  context 'with a Valkyrie resource' do
+    subject(:service) { described_class }
+    let(:object) { FactoryBot.build(:hyrax_resource) }
```

Comment:
Should this be changed at some time to be ':valkyrie_resource` and the other to be `:af_resource`? Nothing needs to be changed here, just an idea for clarity.

Reply:
i think the naming is aimed at being resilient to the eventual removal of AF/Wings. `:hyrax_resource` seems like a reasonable name for the factory that builds `Hyrax::Resource` objects? i would expect `:valkyrie_resource` to build a `Valkyrie::Resource`, i think